[Alerts][Docs] Added API documentation for alerts plugin (#91067)
elastic/kibana: docs/api/alerts/mute_all.asciidoc

[[alerts-api-mute-all]]
=== Mute all alert instances API
++++
<titleabbrev>Mute all alert instances</titleabbrev>
++++

Mute all {kib} alert instances.

[[alerts-api-mute-all-request]]
==== Request

`POST <kibana host>:<port>/api/alerts/alert/<id>/_mute_all`

[[alerts-api-mute-all-path-params]]
==== Path parameters

`id`::

>   (Required, string) The alert ID that you want to mute all instances.

Reviewer: ```suggestion
  (Required, string) The Alert ID of the Alert whose instances we wish to mute.
```
